src/test/java/teammates/test/pageobjects/AdminEmailPage.java

```diff
@@ -20,6 +23,18 @@ public void inputRecipient(String recipient) {
         recipientBox.sendKeys(recipient);
     }
 
+    public void inputGroupRecipient(String fileName) {
+        WebElement groupReceiverListUploadBox = this.getGroupReceiverListUploadBox();
+        JavascriptExecutor js = (JavascriptExecutor) browser.driver;
```

Comment:
Take a look at `Object executeScript(String script, Object... args)` method in `AppPage`